code/modules/clothing/head/misc.dm

```diff
@@ -426,7 +426,7 @@
 	icon_state = "kippah"
 
 /obj/item/clothing/head/medievaljewhat
-	name = "medieval Jew hat"
+	name = "medieval Jewish hat"

```

Comment:
this is the only one that strikes me as a potential ToS violation, the rest are not attacking anything